src/averageAges.js

```diff
@@ -38,6 +48,16 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+
+  const womenAges = withChildren
+    ? people.filter(female => people.find(child => child.mother === female.name))
```

Comment:
Your code seems to be failing the line test. You can move the code after the arrow to a new line.